app/src/components/market/steps/items/resume_market_step.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import moment from 'moment'
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+import { Link } from 'react-router-dom'
+import { APP_URL } from '../../../../common/constants'
+import { LinkSpan } from '../../../common/link_span'
+
+interface Props {
+  values: {
+    question: string
+    category: string
+    resolution: Date | null
+    spread: string
+    funding: string
+    outcomeValueOne: string
+    outcomeValueTwo: string
+    outcomeProbabilityOne: string
+    outcomeProbabilityTwo: string
+  }
+  marketMakerAddress: string | null
+}
+
+class ResumeMarketStep extends Component<Props> {
+  render() {
+    const { marketMakerAddress, values } = this.props
+    const {
+      question,
+      category,
+      resolution,
+      spread,
+      funding,
+      outcomeValueOne,
+      outcomeValueTwo,
+      outcomeProbabilityOne,
+      outcomeProbabilityTwo,
+    } = values
+
+    const resolutionDate = resolution && moment(resolution).format('MM-DD-YYYY')
+
+    const marketMakerURL = `${APP_URL}view/${marketMakerAddress}`
```

Comment:
I don't think we should get this from an environment variable. There must be a way to do it either with `window.location` or with `react-router` (`window.location` might not work if the app is deployed under a sub-path).

Reply:
React router doesn't have a solution for this, I use window location